source/speechDictHandler/__init__.py

```diff
@@ -118,6 +118,14 @@ def sub(self, text):
 def processText(text):
 	if not globalVars.speechDictionaryProcessing:
 		return text
+	try:
+		# see issue 14689
+		if config.conf["UIA"]["winConsoleImplementation"] == "UIA":

```

Comment:
This will fail if the config setting is `auto`. Can you check the API level of the console instead?

Reply:
But if config setting is auto UIA is not used at this API Level, and no freeze happen. Or do you mean that blank lines are present also in API Level>0 when UIA is automatically used?

Reply:
Blank lines are present before microsoft/terminal#6986 was merged, so API level <= `IMPROVED` I believe. I'd really prefer this be checked relative to the API level, not the config.

Reply:
Ok, I have a better overview now. I'll try to check for apiLevel < 2.